### DISCUSSION TITLE
Error "Unexpected token u in JSON at position 0"

### DISCUSSION THREAD

**Discussion by programmerxx89** [fattureincloud/api#357] [Q&A] [answered]
I get this error connecting fattureincloud to zapier.
what does it mean?

**Comment by valmoz** [comment]
Hi @programmerxx89 
it means that the value passed was not a JSON string, but instead started with a "u" character.

We suggest you contact Customer Support to be able to further investigate.

Thanks
Mauro

**Reply by valmoz** [reply]
Hi @programmerxx89 have you solved this issue? have you contacted our Customer Support on the Fatture in Cloud page?

We've seen a bug on the Zapier page, but we don't know if it was you who opened it. 

Thanks
Mauro

**Reply by valmoz** [reply]
Ok The Customer Support received your request. We're on it.

**Comment by emajo** [comment, marked as answer]
Hi @programmerxx89, we fixed the bug on our zapier integration, now the problem should be resolved.

Thanks,
Emanuele.
